src/loaders/TextureLoader.js

```diff
@@ -19,6 +19,12 @@ Object.assign( THREE.TextureLoader.prototype, {
 		loader.setPath( this.path );
 		loader.load( url, function ( image ) {
 
+			// JPEGs can't have an alpha channel, so memory can be saved by storing them as RGB.
+			var isJPEG = url.search(/\.(jpg|jpeg)$/) > 0;
+			if (!isJPEG) {
+				isJPEG = url.search(/^data\:image\/jpeg/) == 0;
```

Comment:
It's data URLs that have the image/jpeg type, something like this:

data:image/jpeg;base64,/9j/4AAQSkZJRgA...